test/test_YoutubeDL.py

```diff
@@ -1252,7 +1252,7 @@ def _test():
                     _test()
 
         test('test=value; Domain=.yt.dlp', [cookie('test', 'value', domain='.yt.dlp')])
-        test('test=value', [cookie('test', 'value')], error='Unscoped cookies are not allowed')
+        test('test=value', [cookie('test', 'value')], error=r'Unscoped cookies are not allowed')
```

Comment:
Why? Is it just too late at night? (and below)
```suggestion
        test('test=value', [cookie('test', 'value')], error='Unscoped cookies are not allowed')
```

Reply:
To make it clear that it is a regex

Reply:
Yes, but that's not what the `r` prefix means, is it? It just means that `\` is literal in the string and there aren't any.